v2/product/product-type.rst

```diff
@@ -1,4 +1,68 @@
 Create a product type
 =====================
+For this example we will create a product type for T-shirts
+## First create Product attribute
+First we will need to create a product attribute. We will create two product attributes - 
 
-@todo
+* Size
+* Color
+
+Goto ``admin/commerce/product-attributes``
```

Comment:
For important menu transitions, you should explain, especially to customers, where they can find this in the menu system. They don't know what URLs are.